### PR TITLE
LPS-146379 Only add the export translation action if it can be shown

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-146379 only add to available actions the export translation if can be shown
- LPS-146379 use the methods from layoutsAdminDisplayContext to delete repeated code

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java [hunk 1]
```diff
@@ -1713,6 +1722,17 @@ public boolean isShowViewLayoutAction(Layout layout) {
...
+		Set<Locale> availableLocales = LanguageUtil.getAvailableLocales(
+			themeDisplay.getSiteGroupId());
+
+		if (availableLocales.size() == 1) {
+			return true;
+		}
+
+		return false;
+	}
+
```
Comment (adolfopa) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java:1735]: It seems this isn't use outside of the class. Can we make it private?

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Other than the small thing I commented about above this looks good. Forward directly when you're done with this.

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/LayoutsAdminDisplayContext.java:1735]: It's used on 
`LayoutActionDropdownItemsProvider#469` and` LayoutActionDropdownItemsProvider#486`

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**